Assets/HoloToolkit/SpatialSound/Scripts/AudioLoFiEffect.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// An audio effect that limits the frequency range of a sound to simulate
+    /// being played over various telephony or radio sources.
+    /// </summary>
+    /// <remarks>
+    /// For the best results, also attach an AudioInfluencerManager to the sound
+    /// source. This will ensure that the proper frequencies will be restored
+    /// when audio influencers are used in the scene.
+    /// </remarks>
+    [RequireComponent(typeof(AudioSource))]
+    [RequireComponent(typeof(AudioLowPassFilter))]
+    [RequireComponent(typeof(AudioHighPassFilter))]
+    [DisallowMultipleComponent]
+    public class AudioLoFiEffect : MonoBehaviour
+    {
+        /// <summary>
+        /// The quality level of the simulated audio source (ex: AM radio).
+        /// </summary>
+        [Tooltip("The quality level of the simulated audio source.")]
+        public AudioSourceQuality SourceQuality;
+
+        // The audio filter settings that match the selected source quality.
+        private FilterSettings filterSettings;
+
+        // The filters used to simulate the source quality.
+        private AudioLowPassFilter lowPassFilter;
+        private AudioHighPassFilter highPassFilter;
+
+        // Collection used to look up the filter settings that match the selected
+        // source quality.
```

Comment:
let's use the summary tags here

Reply:
Are we using summary tags for privates as well as publics? Will the auto documentation tool do the correct filtering?

Reply:
I guess that depends on the tool we use.

Reply:
In either case I'd like to at least inspect it and see a description via tooltip

Reply:
Summary tags were added throughout.

Reply:
Thanks!